src/python/pants/backend/python/target_types.py

```diff
@@ -254,10 +255,39 @@ def value_or_global_default(self, pex_binary_defaults: PexBinaryDefaults) -> boo
         return self.value
 
 
+class PexExecutionMode(Enum):
+    ZIPAPP = "zipapp"
+    UNZIP = "unzip"
+    VENV = "venv"
+
+
+class PexExecutionModeField(StringField):
+    alias = "execution_mode"
+    valid_choices = PexExecutionMode
+    expected_type = str
+    help = (
+        "The mode the generated PEX file will run in. The traditional PEX file runs in "
```

Comment:
Formatting nit:

```suggestion
        "The mode the generated PEX file will run in.\n\nThe traditional PEX file runs in "
```

Reply:
Will do. This seems very error prone though. If the goal is to enforce some sort of layout it would seem better to have help take a `Help` which was something like:
```python
@dataclass(frozen=True)
class Help:
  summary: str
  paragraphs: List[str]
```
Relying on a review hawkeye will fall down fast.

Reply:
That's a great suggestion! My first reaction was concern if we could not handle this case anymore:

```
A list of options

  * opt 1
  * opt 2

More text.
```

Where each bullet is separate by a single `\n`. But that's easy to support..the list is a single "paragraph", with a single `\n` between each sentence. Paragraphs can still have arbitrary `\n`s in them; only, we'll now use `"\n\n".join()` over the paragraphs. 

--

Imo, we'd want to do this to the options system too. We've been encouraging using newlines there too to make help messages more readable.

Perhaps we can rename all these to `help_paragraphs`? That would be an excellent starter issue for Liam.

Reply:
Fixed.